visallo/visallo
# Histogram should handle PVs with many values better

We don't currently limit how many possible values are shown in the histogram. This PR will make Visallo handle PVs like all other datatypes.

![screen shot 2017-12-07 at 10 12 30 am](https://user-images.githubusercontent.com/152878/33722154-33eb7c9a-db37-11e7-9dcf-b08191460a85.png)


Testing Instructions:
Select more than 5 entities on the graph, each of which should have a possible value property set to a different value. The histogram should not show more than five bars for that property.
